examples/js/loaders/gltf/glTFLoader.js

```diff
@@ -1547,8 +1573,8 @@ THREE.glTFLoader.prototype.load = function( url, callback ) {
 											boneInverses, false ), skin.bindShapeMatrix );
 
 										//threeMesh.bindMode = "detached";
-										//threeMesh.normalizeSkinWeights();
-										//threeMesh.pose();
+										threeMesh.normalizeSkinWeights();
+										threeMesh.pose();
```

Comment:
Actually, it is probably best to not change these lines :) Fixed in 1afc78a940640e4fb85ce72879253c9a91114cb0.